src/views/PositionPdfPlot/PositionPdfPlotWidget.tsx

```diff
@@ -177,22 +173,23 @@ const PositionPdfPlotWidget: FunctionComponent<Props> = ({fetchSegment, startTim
                 c.fillRect(xx - 0.5, yy + 0.5, 1, 1)
             }
         }
-        context.save()
-        context.scale((pixelTimes[1] - pixelTimes[0]) / imageData.width, panelHeight / imageData.height)
-        context.drawImage(canvas, pixelTimes[0], 0)
-        context.restore()
-    }, [pixelTimes, panelHeight, imageData, visibleLinearPositions, showLinearPositionsOverlay, downsampleFactor])
+        // Draw scaled version of image
+        // See: https://stackoverflow.com/questions/3448347/how-to-scale-an-imagedata-in-html-canvas
+
+        // Scaling the offscreen canvas can be done when it's drawn in, which avoids having to deal with transforms and some margin issues.
+        context.clearRect(0, 0, context.canvas.width, context.canvas.height)
+        context.drawImage(canvas, 0, 0, panelWidth, panelHeight)
```

Comment:
We can apply a scaling operation to the bitmap being brought in as part of the `drawImage` call--no need to touch the (onscreen) canvas's transform.